src/utils/sorting.ts

```diff
@@ -11,3 +13,16 @@ export const sortData = (data, prop) => {
 
   return myData;
 };
+
+export const sortGeos = (data, prop) => {
+  var myData = data.sort((a, b) => {
+    if (b[prop] == "other") return -1;
```

Comment:
Use `===` please, double equals is generally not recommended anymore

Reply:
Damn python indoctrination!